### DISCUSSION TITLE
park-interaction error

### DISCUSSION THREAD

**Discussion by MauricioOlitel** [twilio-professional-services/flex-project-template#455] [Q&A]
The list of parked calls has a limit, when the agent pauses several calls the system returns an error and the list disappears. Any one has the same problem and found an solution for it ?

**Comment by dremin** [comment]
Could you please provide the error you are receiving, and after how many items are you receiving the error?

**Comment by MauricioOlitel** [comment]
![error_twilio](https://github.com/twilio-professional-services/flex-project-template/assets/142459430/dd1b0214-1864-48b1-ba00-21d0be88da38)
Hi,
Attached I am sending a printout of the error, the client was unable to give me an exact number, he believes that more than 30, one thing that would be interesting is to add a scroll bar.

**Reply by dremin** [reply]
Hi @MauricioOlitel, sorry for the delay on this. Thanks for the picture, I believe I see the issue and will get a PR up with a fix.

**Comment by dremin** [comment]
This was fixed in #478.
